packages/console-app2/src/components/ResourceList.tsx

```diff
@@ -38,8 +37,8 @@ function ResourceList ({ config }) {
       <List>
         {(apps ?? []).map(app => {
```

Comment:
Don't be defensive here -- assume that the variable is well formed (by default value). Just `apps.map`

Reply:
👍